### ISSUE TITLE
Add file descriptor usage

### ISSUE THREAD

**Issue by SoyYoRafa** [scylladb/scylla-monitoring#726]
Hi,
Could you please add monitoring for file descriptor usage per instance? It's a resource that can be exhausted and it would be good how much head room there is.

**Comment by tzach** [comment]
node_exporter expose file descriptor info with `filefd`[1]

[1] https://github.com/prometheus/node_exporter/blob/master/collector/filefd_linux.go

**Comment by amnonh** [comment]
The relevant metrics are:
`node_filefd_allocated` and `node_filefd_maximum`

**Comment by SoyYoRafa** [comment]
These attributes collect the kernel limits. Can we also collect the scylla process limit `ulimit -n`? The scylla process would have to export this.

**Comment by amnonh** [comment]
@tzach is there a point in showing the kernel limit or usage?

**Comment by tzach** [comment]
Yes, as they might run out, but I'm not sure it's as interesting.
@SoyYoRafa please note you can add you own custom metrics

**Comment by fee-mendes** [comment]
@amnonh bump

**Comment by amnonh** [comment]
depends on #2366
